arm-asazure/2016-05-06-preview/swagger/asazure.json

```diff
@@ -0,0 +1,489 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-05-06-preview",
+    "title": "ASAzureWebAPI",
+    "description": "The Azure Analysis Services API provides a RESTful set of web services enables users to create, retrieve, update, and delete servers"
+  },
+  "host": "management.azure.com",
+  "basePath": "/webapi",
+  "schemes": [ "https" ],
+  "paths": {
+    "/resourceprovider/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers/{analysisServerName}": {
+      "get": {
+        "description": "Gets the current state of the specified Analysis Services server",
+        "tags": [ "RpProvision" ],
+        "operationId": "RpProvision_GetDetailsAsync",
+        "consumes": [ "application/json", "text/json"],
+        "produces": [ "application/json", "text/json"],
+        "parameters": [
+          {
+            "name": "analysisServerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "name": "cancellationToken",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/CancellationToken" },
+            "description": "Cancellation token"
+          },
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Subscription Id for the subscription in which the operation is executed"
```

Comment:
SubscriptionId should be defined in the global parameters section and should be referenced everywhere. All globally defined parameters will become client properties and will not appear as method parameters. That provides a better user experience to our customers.

Please take a look at any other spec in this repo.